docs/docs/running-offen/configuring-the-application.md

```diff
@@ -211,6 +211,13 @@ Default value `no-reply@offen.dev`.
 
 The From address used when sending transactional email.
 
+### OFFEN_SMTP_AUTHTYPE
+{: .no_toc }
+
+Default value `LOGIN`.
+
+The SMTP authentication type used when sending transactional email. Supported types are: `LOGIN`, `PLAIN`, `CRAM-MD5`, `NOAUTH`.
```

Comment:
As the code now rejects `NOAUTH` I wonder if we should omit it here? Any specific reason you included it in the list?

Reply:
I've snuck in support for "noauth" with commit https://github.com/offen/offen/pull/827/commits/cdcb24df615089e667650db4e8dddc95b59b3a19. I thought it might be useful in some enterprise settings.

Reply:
Sorry, I forgot that Go does not fallthrough to default, but this means it instead does nothing. Nevermind.